kaiax/auction/impl/getter.go

```diff
@@ -42,12 +38,20 @@ func (a *AuctionModule) GetBidTxGenerator(tx *types.Transaction, bid *auction.Bi
 			return nil, err
 		}
 
+		if bid.GetGasLimit() == 0 {
+			gasLimit, err := a.bidPool.getBidTxGasLimit(bid)
+			if err != nil {
+				return nil, err
+			}
+			bid.SetGasLimit(gasLimit)
+		}
```

Comment:
Locating the gas setting here again seems not to harm, but just curiosity, is there a case when a bid's gas has not been set at this point?

Reply:
No such case exists. It's for just in case.